spanner/spec/json_column_type_spec.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "../json_column_type_samples"
+require_relative "./spec_helper"
+
+describe "Spanner database JSON datatype" do
+  def create_test_database
+    statements = [
+      "CREATE TABLE Venues (
+        VenueId INT64 NOT NULL,
+      ) PRIMARY KEY(VenueId)"
+    ]
+
+    job = db_admin_client.create_database \
+      parent: instance_path(@instance_id),
+      create_statement: "CREATE DATABASE `#{@database_id}`",
+      extra_statements: statements
+
+    job.wait_until_done!
+    job.results
+  end
+
+  before :each do
+    cleanup_database_resources
+    create_test_database
+  end
+
+  after :each do
+    cleanup_database_resources
+  end
+
+  example "add_json_column" do
```

Comment:
Do we need to merge three examples into a single one? I feel we are doing the same thing three times, e.g., `add_json_column`. We can just have the last test case and verify after each step.

Reply:
merged.